### DISCUSSION TITLE
Sentry tags high cardinality

### DISCUSSION THREAD

**Discussion by tommy-ng0** [getsentry/sentry-php#1753] [Q&A] [answered]
Hi there,

I am coming from Prometheus/VictoriaMetrics world so dynamic tags are a big no no. Does Sentry also avoid having dynamic tags? I couldn't find anything when searching for high cardinality tags.

For more context, I want my JobID which is uuid to be searchable in Sentry (each JobID is unique for each job so it will only happen once). Seems like the only way to do this is by tagging. I am wondering if there is a better way or is tagging the only way to go and high cardinality is not a thing?

**Comment by cleptric** [comment, marked as answer]
It’s fine to attach the job id as a tag.
